pkg/cmd/run/run.go

```diff
@@ -75,6 +78,7 @@ func NewCmdRun(o *Options) *cobra.Command {
 	bootstrapFlags := namedFlagSets.FlagSet("bootstrap")
 	bootstrapFlags.BoolVar(&o.BootstrapCRDs, "crd", true, "if set, the operator will attempt to install/update the CRDs before starting up.")
 	bootstrapFlags.StringVar(&o.BootstrapSpicedbsPath, "bootstrap-spicedbs", "", "set a path to a config file for spicedbs to load on start up.")
+	bootstrapFlags.StringVar(&o.WatchNamespaces, "watch-namespaces", "", "set a comma-separated list of namespaces to watch for CRDs.")
```

Comment:
A [StringSliceVar](https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar) would be better here, since it will take care of parsing out commas for you.